RFCs/FS-1112-obsolete-allowed-to-use-obsolete.md

```diff
@@ -0,0 +1,114 @@
+# F# RFC FS-1112 - Obsolete allowed to use Obsolete
+
+<!--The design suggestion [Obsolete allowed to use Obsolete](https://github.com/fsharp/fslang-suggestions/issues/1055) has been marked "approved in principle".
+-->
+
+This RFC covers the detailed proposal for this suggestion.
+
+- [x] [Suggestion](https://github.com/fsharp/fslang-suggestions/issues/1055)
+- [ ] Approved in principle
+- [ ] [Implementation](https://github.com/dotnet/fsharp/pull/FILL-ME-IN)
+- [ ] Design Review Meeting(s) with @dsyme and others invitees
+- [Discussion](https://github.com/fsharp/fslang-design/discussions/FILL-ME-IN)
+
+# Summary
+
+The general idea is to allow types and functions marked with `System.ObsoleteAttribute` to use other `Obsolete`-marked members without
+having the FS0044 warning emitted. 
+
+# Motivation
+
+Assume we completely redesigned API of some library. Then, to keep the backward compatibility we keep the old members and mark
+them with `Obsolete`. But if there was no equivalent to old functions in the new library, we have to use obsolete functions
+and members in that old obsolete API. Since the member already marked as `Obsolete`, and hence, unsupported, there is no reason
```

Comment:
```suggestion
and members in that old obsolete API. Since the member is already marked as `Obsolete`, and hence, unsupported, there is no reason
```